src/ol_infrastructure/substructure/keycloak/__main__.py

```diff
@@ -116,3 +116,15 @@
     enabled=True,
     opts=resource_options,
 )
+
+# Create Dagster OIDC client
+keycloak_domain_name = keycloak_config.require("url")
```

Comment:
Should we make this a `StackReference`?

Reply:
I went with this cause I noticed we had a comment on line 12 about issues with pulumi reading stack defiinition

Reply:
I think those are disparate issues, but this works for today. We can revisit later.